test/core/CoreTest.cpp

```diff
@@ -7,6 +7,19 @@ namespace {
 TEST(CoreTest, itLives)
 {
     Core();
+}
+
+TEST(CoreTest, changeProductQuantityShouldReturn2)

```

Comment:
spróbuj storzyć klase testową, i użyć zamisat TEST makra TEST_F

Coś takiego, nie jestem pewny czy to się skompiluje, bo pisze z pamięci

class CoreTest : public ::testing::Test
{
public:
Core core;
}

TEST_F(CoreTest, changeProductQuantityShouldReturn2)
{
// teraz pole Core core jest widoczne w każdej TEST_F, która korzysta z klasy CoreTest
// dlatego nie musisz za każdym razem deklarować Core core, tylko wystarczy ta linijka
EXPECT_EQ(core.changeProductQuantity(0,"",1), 2);
}